lib/modules/manager/bitbucket-pipelines/extract.spec.ts

```diff
@@ -29,15 +29,23 @@ describe('modules/manager/bitbucket-pipelines/extract', () => {
             "depType": "docker",
             "replaceString": "node:10.15.2",
           },
+          {
+            "autoReplaceStringTemplate": "{{depName}}{{#if newValue}}:{{newValue}}{{/if}}{{#if newDigest}}@{{newDigest}}{{/if}}",
+            "currentDigest": undefined,
+            "currentValue": "2.0.2",
+            "datasource": "docker",
+            "depName": "jfrogecosystem/jfrog-setup-cli",
+            "depType": "docker",
+            "replaceString": "jfrogecosystem/jfrog-setup-cli:2.0.2",
+          },
           {
             "currentValue": "0.2.1",
             "datasource": "bitbucket-tags",
             "depName": "atlassian/aws-s3-deploy",
-            "depType": "bitbucket-tags",
```

Comment:
Needs reverting?

Reply:
haha - ok - i flipped a coin and interpreted your rollback comment literally :p.  Take two coming up

Reply:
Third times a charm 🤞 🤞